ddl/stat_test.go

```diff
@@ -140,14 +144,30 @@ func TestGetDDLInfo(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func addDDLJobs(txn kv.Transaction, job *model.Job) error {
+func addDDLJobs(sess session.Session, txn kv.Transaction, job *model.Job) error {
+	if variable.EnableConcurrentDDL.Load() {
+		b, err := job.Encode(true)
+		if err != nil {
+			return err
+		}
+		_, err = sess.Execute(context.Background(), fmt.Sprintf("insert into mysql.tidb_ddl_job(job_id, reorg, schema_ids, table_ids, job_meta, type, processing) values (%d, %t, %s, %s, %s, %d, %t)",
+			job.ID, job.MayNeedReorg(), strconv.Quote(strconv.FormatInt(job.SchemaID, 10)), strconv.Quote(strconv.FormatInt(job.TableID, 10)), wrapKey2String(b), job.Type, false))
+		return err
+	}
 	m := meta.NewMeta(txn)
 	if job.MayNeedReorg() {
 		return m.EnQueueDDLJob(job, meta.AddIndexJobListKey)
 	}
 	return m.EnQueueDDLJob(job)
 }
 
+func wrapKey2String(key []byte) string {
```

Comment:
This is duplicated with https://github.com/pingcap/tidb/pull/32169/commits/e05c563e51691f6246fa4295764a6ea75ff40eba#diff-91c692cc214077acc788fcaf2f92fb40d0a6c4aef3e34e3b6ed532677dac9f13R303？

Reply:
yes, they are the same. Because we have no public function to add a job to the table, I write it manually and get some duplicated code